docs/guide.rst

```diff
@@ -196,31 +196,116 @@ the scene as desired::
 data that defines (the shape of) the object, such as positions, plus
 data associated with these positions (normals, texcoords, colors, etc.).
 Multiple world objects may share a geometry.
 
 
-Materials
----------
+**Materials**
 
-Each world object also has a material. This material object defines the
+All visible world objects have a material. This material object defines the
 appearance of the object. Examples can be its color, how it behaves under lighting,
 what render-mode is applied, etc. Multiple world objects may share a material.
 
 
+Cameras and controllers
+-----------------------
+
+We've already been using cameras, but let's look at them a bit closer!
+
+
+**Perspective camera**
+
+There are two main cameras of interest. The first is the :class:`~pygfx.cameras.PerspectiveCamera`,
+which is a generic camera intended for 3D content. You can instantiate one
+like this:
+
+.. code-block:: python
+
+    camera = gfx.PerspectiveCamera(50, 4/3)
+
+The first argument is the fov (field of view) in degrees. This is a value between 0 and 179,
+with typical values < 100.
+
+The second argument is the aspect ratio. A window on screen is usually a rectangle
+with 4/3 or 16/9 aspect. The aspect can be set so the contents better fit the window.
+
+When the fov is zero, the camera operates in orthographic mode, where there is
+no perspective. In this case it'd be good to also specify a 3d argument: the extent
+specifies the size of the thing being viewed. So if your scene is about 400 units in size:
+
+.. code-block:: python
+
+    camera = gfx.PerspectiveCamera(0, 1, 400)
+
+
+**Orthographic camera**
+
+The second camera of interest is the :class:`~pygfx.cameras.OrthographicCamera`. It's similar
+to the perspective camera, except it's fov is always zero. It can be instantiated like this:
+
+.. code-block:: python
+
+    camera = gfx.OrthographicCamera(400, 400, maintain_aspect=False)
+
+
+The two first arguments are the `width` and `height`, which are
+typically used to initialize an orthographic camera. They have a direct
+relation with aspect and extent. (The extent is their mean, and aspect their ratio.)
+
+The `maintain_aspect` argument can be set to False if the dimensions
+do not represent a physical dimension, e.g. for plotting data. The contents
+of the view are then stretched to fill the window.
+
+
+**Orienting cameras**
+
+Camera's can be oriented manually by setting their position, and then set their rotation
+to have them look in the correct direction. This last bit can be tricky, unless you use
+the :func:`.look_at()<pygfx.cameras.PerspectiveCamera.look_at>` method. This also sets
+the extend (based on what's visible at the target location), so you can now attach
+a controller and interact!
+
+Even easier is using :func:`.show_object()<pygfx.cameras.PerspectiveCamera.show_object>`.
+With this method you specify an object (e.g. the scene) and optionally a direction,
```

Comment:
It would make more sense for this method to adjust the near and far plane (assuming that's what "extent" means)